WalletWasabi.Fluent/ViewModels/Wallets/CoinJoinStateViewModel.cs

```diff
@@ -103,6 +104,9 @@ public CoinJoinStateViewModel(WalletViewModel walletVm, IObservable<Unit> balanc
 			.ObserveOn(RxApp.MainThreadScheduler)
 			.Subscribe(StatusChanged);
 
+		this.WhenAnyValue(x => x.RemainingTime)
+			.Subscribe(text => IsBalanceDisplayed = text.Contains("BTC"));
```

Comment:
What if in the future we add an option to display amounts in sats?

Reply:
It would not work. But while we are storing a bitcoin value in a variable called `RemainingTime` I don't care about it.

The current statement about MusicBox is to make it work and stable as soon as possible. Then someone can start refactoring it.